libs/components/forms/src/lib/modules/error/errors.component.ts

```diff
@@ -0,0 +1,56 @@
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+import { CommonModule } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+} from '@angular/core';
+import { ValidationErrors } from '@angular/forms';
+import { SkyIdModule } from '@skyux/core';
+import { SkyI18nModule } from '@skyux/i18n';
+
+import { SkyFormsResourcesModule } from '../shared/sky-forms-resources.module';
+
+import { SkyErrorComponent } from './error.component';
+
+/**
+ * @internal
+ */
+@Component({
+  selector: 'sky-errors',
+  standalone: true,
+  imports: [
+    CommonModule,
+    SkyI18nModule,
```

Comment:
The `SkyI18nModule` should be exported by `SkyFormsResourcesModule` so is redundant.

Reply:
Yes, good call, updated.